File: JobOpportunities.Domain/Users/CompanyAgent.cs
namespace JobOpportunities.Domain.Users
{
    public class CompanyAgent : ApplicationUser
    {
        public CompanyAgent(string firstName, string lastName, string cuit) : base(firstName, lastName)
        {
            Cuit = cuit;
        }

        public string Cuit { get; set; }
        public ICollection<JobOffer>? Offers { get; set; }

    }
}

File: JobOpportunities.SPA/src/app/common/models/company-agent.ts
export interface CompanyAgent {
  cuit: string;
  offers: Offer[];
  id: string;
  userName: any;
  email: any;
  firstName: string;
  lastName: string;
}

export interface Offer {
  id: string;
  title: string;
  description: string;
  validUntil: Date;
}
